cli/tests/integrations/test_package.py

```diff
@@ -75,7 +75,11 @@ def test_repo_list():
         ).format(**UNIVERSE_TEST_REPOS),
         'utf-8'
     )
-    assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
+
+    # test overriding cosmos_url config
+    dcos_url = config.get_config_val("core.dcos_url")
+    with update_config('package.cosmos_url', dcos_url, env):
+        assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
```

Comment:
If it's an integration test, it will have to test something else as well. But that's why I added a comment.